firmware/brake/kia_soul_ev/src/init.cpp

```diff
@@ -51,4 +52,10 @@ void init_communication_interfaces( void )
 
     DEBUG_PRINT( "init Control CAN - " );
     init_can( g_control_can );
+
+    // Filter CAN messages - accept if (CAN_ID & mask) == (filter & mask)
+    g_control_can.init_Mask( 0, 0, 0x7F0 ); // Filter for 0x0N0 to 0x0NF
+    g_control_can.init_Mask( 1, 0, 0x7FF ); // Don't use second filter
```

Comment:
If we're not using the second filter, why are we initializing it? The mask `0x7FF` should require an exact match with the associated filtered CAN ID, is that the intent here?

Reply:
If you don't do that step then the mask lets everything in and makes the other mask useless.

Reply:
mask 0 is for buffer 0 and mask 1 is for buffer 1. Data sheet and mcp library shows buffer 1 fills when buffer 0 hits overflow. For our case since we have a limited number of messages of equal importance we should probably set the same filter on both. Since the overflow status isn't supported in the mcp library we cannot error on buffer 1 overflow.